imports/ui/components/home/home.html

```diff
@@ -11,8 +14,13 @@
       </div>
     {{/each}}
   </div>
-  {{#if reviews}}<h4 class='home__headline'>Latest Reviews</h4>{{/if}}
-  <div class='home__reviews'>
+</div>
+<div class='home__reviews white'>
```

Comment:
My preference would be not to have a 'black' and 'white' class. Maybe I setup expectations otherwise, though, saying it would be good to have reusable CSS. Anyway, no big deal if this stays as it is for now.